zaza/openstack/utilities/openstack.py

```diff
@@ -2225,6 +2225,19 @@ def get_images_by_name(glance, image_name):
     return [i for i in glance.images.list() if image_name == i.name]
 
 
+def get_volumes_by_name(cinder, volume_name):
+    """Get all cinder volume objects with the given name.
+
+    :param cinder: Authenticated cinderclient
+    :type cinder: cinderclient.Client
+    :param image_name: Name of volume
+    :type image_name: str
+    :returns: List of cinder volumes
+    :rtype: [cinderclient.v3.volume, ...]
```

Comment:
Technically, the rtype should be `List[cinderclient.v3.volume]`

Reply:
Done